### PR TITLE
Classify 'var' in patterns as a keyword.

### PR DESCRIPTION
**Pull request by CyrusNajmabadi**
Fixes https://github.com/dotnet/roslyn/issues/18956

### CODE REVIEW

**Comment by CyrusNajmabadi** [comment]
Tagging @dotnet/roslyn-ide

File: src/EditorFeatures/Core/Implementation/Suggestions/SuggestedActionsSource.cs [hunk 1]
```diff
@@ -49,7 +49,6 @@ public SuggestedActionsSource(SuggestedActionsSourceProvider owner, ITextView te
                 _owner = owner;
                 _textView = textView;
                 _textView.Closed += OnTextViewClosed;
-
```
Comment (Pilchie) [inline src/EditorFeatures/Core/Implementation/Suggestions/SuggestedActionsSource.cs]: Not part of this PR.

**Comment by CyrusNajmabadi** [comment]
Tagging @MattGertz 

This is a minor, but very visible classification issue in the IDE where we don't end up classifying "var" properly as a keyword.  

**Customer scenario**

Customer uses the new 'var' pattern match construct.  IDE classification is not consistent and is annoyingly inconsistent.

**Bugs this fixes:**

#18956

**Workarounds, if any**

None.

**Risk**

Low.

**Performance impact**

Low.

**Is this a regression from a previous update?**

No.  This ia new feature.

**Root cause analysis:**

Insufficient testing of all IDE features against all new language constructs.

**How was the bug found?**

Internal customer reported.
